cg/services/run_devices/pacbio/data_transfer_service/data_transfer_service.py

```diff
@@ -37,9 +38,9 @@ def get_post_processing_dtos(self, run_data: PacBioRunData) -> PacBioDTOs:
         sequencing_run_dto: PacBioSequencingRunDTO = get_sequencing_run_dto(
             metrics=metrics, run_data=run_data
         )
-        sample_sequencing_metrics_dtos: list[PacBioSampleSequencingMetricsDTO] = (
-            get_sample_sequencing_metrics_dtos(metrics.samples)
-        )
+        sample_sequencing_metrics_dtos: list[
```

Comment:
linting 👀

Reply:
I mean, this is what my black does - v24.8.0 line length 100